docs/src/main/sphinx/release/release-407.md

```diff
@@ -8,37 +8,96 @@
   `query.remote-task.request-size-headroom`, and
   `query.remote-task.guaranteed-splits-per-task` configuration properties or
   their respective session properties. ({issue}`15721`)
+* Add support for correlated queries with single-row values. ({issue}`15989`)
+* Improve query memory tracking. ({issue}`15983`)
 * Improve query performance when a predicate evaluates to a null value. ({issue}`15744`)
 * Improve performance of queries with joins on the output of global
   aggregations. ({issue}`15858`)
-* Fix precision loss when converting `time` values with a precision higher than
-  three and `time with time zone` values with lower precision. ({issue}`15861`)
+* Improve performance of selective queries, queries that read a small number of
+  columns, and queries that process tables with large Parquet row groups or Orc
+  stripes. ({issue}`15579`)
+* Improve concurrency when small, concurrent queries are run on a large cluster.
+  Trino now automatically determines many nodes to use for distributed joins
+  and aggregations. This is managed by the `query.max-hash-partition-count`
+  (renamed from `query.hash-partition-count`) and `query.min-hash-partition-count`
+  configuration properties. You can also now set the minimum number of rows and
```

Comment:
I am not a friend of using "now" .. especially not twice..

Reply:
Could this even be a separate release notes entry .. but with the same PR number?